cg/meta/orders/ticket_handler.py

```diff
@@ -22,9 +21,13 @@ class TicketHandler:
 
     NEW_LINE = "<br />"
 
-    def __init__(self, osticket_api: OsTicket, freshdesk_client: FreshdeskClient, status_db: Store):
-        self.osticket: OsTicket = osticket_api
-        self.freshdesk_client: FreshdeskClient = freshdesk_client
+    def __init__(self, status_db: Store, base_url: str, api_key: str, order_email_id: int, env: str):
+        self.client=FreshdeskClient(
```

Comment:
How come we are backtracking on passing the client in the constructor and instead create it here?